packages/frontend/components/shared/solution/Solution.tsx

```diff
@@ -7,38 +7,43 @@ import Modal from '../modal/Modal';
 import { Solution } from '@kodifaj/common';
 import Badges from '../../badges/Badges';
 import { useUser } from '../../context/UserContext';
-import SolutionForm from '../solutionForm/SolutionForm';
+import SolutionFormForUser from '../solutionForm/solutionFormForUser/SolutionFormForUser';
```

Comment:
te komponenty nie powinny być w `/shared` - edwidentnie sam napisałeś że jedna jest dla Usera tylko a druga dla Taska

Reply:
przeniosłem do `components` - zastanawiam się czy nie lepiej wydzielić jeszcze folder np. `solutionForm` i tam wrzucić. Jakby co to zmienię.